src/views/work/HushView.vue

```diff
@@ -0,0 +1,190 @@
+          </p>
+          <br />
+          <ul class="colored-bullets">
+            <li>
+              Haptic-feedback-based Ouija board puzzle <SimpleLink href="#ouija">[Jump]</SimpleLink>
+            </li>
+            <li>Drawer & fuse box physics</li>
+            <li>Base Player XR controller</li>
+            <li>Hand animation scripting & pivots</li>
+            <li>
+              Puzzle completions leading to aging and portrait changes
+              <br /><SimpleLink href="#state-changes">[Jump]</SimpleLink>
+            </li>
+          </ul>
+        </VerticalBox>
+        <img
+          :src="getImageUrl('Games', `/hush/HushUpstairs.png`)"
+          alt="A dimly lit library with wooden shelves, a candle-lit desk, and red-patterned wallpaper from the video game 'Hush'"
+        />
+      </FlexBox>
+
+      <hr class="portfolio-hr" />
+
+      <!-- Ouija Board -->
+      <FlexBox>
+        <VerticalBox>
+          <h3 id="ouija">Ouija Board.</h3>
+          <h4>Description.</h4>
+          <p>
+            The Ouija board provides the means of communication between the deceased interns and the
+            player. As the user gets closer to the next letter, the ghost vibrates the planchette
+            with a greater intensity. Through this haptic feedback, users are guided to the correct
+            letters, which are written on the chalkboard.
+          </p>
+          <h4>Challenges.</h4>
+          <p>
+            As the team was utilizing Unity XR's Grab Interactable script, the planchette's
+            interactions posed a challenge because this script removes positional constraints on
+            held objects. To avoid the removal of these necessary constraints, I hid the Grab
+            Interactable object's mesh and replaced it with a replica planchette. From there, I used
+            the positional data of the holding controller and bound the replica planchette on the
+            y-axis within the Ouija board. To provide the sense that the ghosts were keeping the
+            planchette in place, I also implemented a solution where the planchette glides back to
+            its original position if the holding controller goes out-of-bounds (See
+            <SimpleLink href="#source-planchette">PlanchetteInteraction.cs</SimpleLink> under Code).
+            <br />
+            The initial Ouija interaction caused the distance-to-letter haptic feedback to become
+            muddled on letter changes. To provide a break between letters, the haptic feedback is
+            reduced to nothing as each letter is drawn on the chalkboard. This Coroutine allows the
+            next haptic vibration to have a greater impact on the user (See
+            <SimpleLink href="#source-ouija">Ouija.cs</SimpleLink> under Code).
+          </p>
+        </VerticalBox>
+
+        <VerticalBox>
+          <video controls muted>
+            <source
+              :src="getImageUrl('Games', `/hush/HushOuija.mp4`)"
```

Comment:
If it's gonna get videos too, let's rename it to something like getAssetUrl or getMediaUrl.

Reply:
Yes! Renamed to getMediaUrl & imageUtils.ts to mediaUtils.ts